test/core/templaters/jinja_test.py

```diff
@@ -551,14 +553,13 @@ def test__templater_jinja_slice_template(test, result):
 
 
 @pytest.mark.parametrize(
-    "raw_file,templated_file,override_context,result",
```

Comment:
Cleaning up tech debt: The `templated_file` parameter in these test cases has not been used for the past several months (I think since `JinjaTracer` was added.)